assets/src/edit-story/components/library/text/fontPreview.js

```diff
@@ -49,25 +50,32 @@ const Text = styled.span`
   color: ${({ theme }) => theme.colors.fg.v1};
 `;
 
-function FontPreview({ title, fontFamily, ...fontProps }) {
+function FontPreview({ title, font, ...rest }) {
```

Comment:
If `rest` is not used anywhere anymore, why not just add `fontSize` and `fontWeight` here and remove `rest`?